### PR TITLE
Split messages to save in cassandra into fixed and configurable parti…

### PR DESCRIPTION
**Pull request by josepablofernandez**
…tions

### CODE REVIEW

File: engine/src/main/config/config.conf [hunk 1]
```diff
@@ -55,6 +55,8 @@ spark = {
 
 cassandra = {
 	hosts = ["localhost:9042"]
+	maxBatchSize = 50
+	batchType = "UNLOGGED"
```
Comment (aargomaniz) [inline engine/src/main/config/config.conf]: I would add a comment defining the different type of batchType values allowed

File: engine/src/test/java/com/stratio/decision/functions/FunctionsTestIT.java [hunk 2]
```diff
@@ -115,7 +116,7 @@ public void testSaveToCassandra() throws Exception {
         LOGGER.debug("Connecting to Cassandra Quorum: " + conf.getStringList("cassandra.hosts").toString());
 
         SaveToCassandraActionExecutionFunction func = new SaveToCassandraActionExecutionFunction(
-                getHostsStringFromList(conf.getStringList("cassandra.hosts")), ProtocolOptions.DEFAULT_PORT);
+                getHostsStringFromList(conf.getStringList("cassandra.hosts")), ProtocolOptions.DEFAULT_PORT, 50,  BatchStatement.Type.UNLOGGED);
```
Comment (aargomaniz) [inline engine/src/test/java/com/stratio/decision/functions/FunctionsTestIT.java]: I would add defaultBatch size as a constant and detail the default value in conf file

**Comment by aargomaniz** [comment]
nice work! Added a cuople of suggestions

File: engine/src/test/java/com/stratio/decision/functions/FunctionsTestIT.java (hunk 2 above)
Comment (josepablofernandez) [inline engine/src/test/java/com/stratio/decision/functions/FunctionsTestIT.java]: There is not a default Batch size value. The configuration must define this value. I added a comment to recommend not to set a value greater than 50

**Pull request merged**